From 63107f515c671c5cde3d67401979705d98949a5c Mon Sep 17 00:00:00 2001
From: Michael W <baddaemon87@gmail.com>
Date: Sun, 11 Mar 2018 11:47:44 +0100
Subject: [PATCH 124/249] wiki: Fix syntax highlighting

* We already have the proper stylesheet, we just don't use it

Change-Id: I1c073dd126dffcce67bc0b8078e6ba77862f2805
---
 _includes/head.html | 1 +
 1 file changed, 1 insertion(+)

diff --git a/_includes/head.html b/_includes/head.html
index 667aa67..66fbeae 100644
--- a/_includes/head.html
+++ b/_includes/head.html
@@ -18,6 +18,7 @@
 <link rel="stylesheet" href="{{ "css/material-kit.css" | relative_url }}">
 <link rel="stylesheet" href="{{ "css/customstyles.css" | relative_url }}">
 <link rel="stylesheet" href="{{ "css/style.css" | relative_url }}">
+<link rel="stylesheet" href="{{ "css/syntax.css" | relative_url }}">
 
 <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700">
 <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
-- 
2.17.1

